Gaijinizer patch/Map048.txt

```diff
@@ -12,41 +12,41 @@ About Beetle Harpy#MTLed
  （男性にとっては脅威である事に変わりませんが）
 ですが『ハーピーモドキ』だけは例外です。
 > CONTEXT: Map048/events/13/pages/0/6/Dialogue
-In an evacuation area that is still far from the war or has
-already left. The demons that appear are mostly harmless.
-(Though they are a potential threat to human males)
-But only Beetle Harpy is an exception.#MTLed
+Demons that appear far from the front lines or in
+already evacuated areas aren't overly dangerous.
+(Though they're still a potential danger to males)
+However, this "Harpy Spawn" is an exception to this.#Ron
 > END STRING
 
 > BEGIN STRING
 この悪魔は鳥類の翼とかぎ爪を持つ女性の姿をしており
 一見ハーピーのような外見をしていますが、
  性格は攻撃的で、人間の血肉を主食としています。
 > CONTEXT: Map048/events/13/pages/0/12/Dialogue
-This demon is in the form of a woman with bird wings and
-claws. It looks like Harpy at first glance,
-It is attack-like in character and is based on human flesh
-and blood.#MTLed
+This demon has the form of a woman with bird wings and
+talons. It has the appearance of a harpy at first
+glance, but it has an aggressive disposition and its
+main diet is human flesh.#Ron
 > END STRING
 
 > BEGIN STRING
 この悪魔たちは、非好戦的な他の悪魔たちとは違って
 非武装の一般市民、特に非力な女性や子供を狙って襲います。
 誘惑なども一切ない、物理的な殺傷です。
 > CONTEXT: Map048/events/13/pages/0/17/Dialogue
-These demons are unlike other non-battle demons
-Attacks unarmed civilians, especially women and children
-who are weak. There is no temptation, it is physical
-killing.#MTLed
+These demons are unlike other non-battle demons.
+They attack unarmed civilians, and particularly target
+helpless women and children. There's absolutely no
+attempt at seduction, only bloodshed.#Ron
 > END STRING
 
 > BEGIN STRING
 ハーピー達を魂の運び手である神の使いとするなら
 ハーピーモドキは、まさしくハイエナ・ハゲタカと呼べるでしょう。
 市民の避難誘導の際は、十分に注意してください。
 > CONTEXT: Map048/events/13/pages/0/22/Dialogue
-If you want Harpy to be the angels of the soul carrier God
-Beetle Harpy can be called a Hyena Vulture.
-In the case of citizen's evacuation guidance, please be
-careful enough.#MTLed
+If harpies are soul-bearing divine messengers,
```

Comment:
It feels like he's making a comparison here, comparing regular Harpies to angels/angelic theming, and my gut wants to compare Harpy Spawn to demons to match, but the base text doesn't seem to support it. X_X

Do you think we should add a comparison like that here anyways?

Edit: Wait.... on second thought..... What if we were to compare regular Harpies to Doves, which would then make sense to compare to Harpy Spawn being Vultures? That makes more sense in my end while reconciling with the original text.

Reply:
That makes a lot of sense. It's a deviation from the base text but a close translation does come across as pretty culturally awkward in English.

I'll leave the phrasing up to you on this one.

Reply:
Hmm, something like this, I think:
>**_If regular Harpies are heavenly doves, 
Harpy Spawn could be called demonic vultures.
Please take great care when evacuating citizens._**